examples/break-statement/break_statement.md

```diff
@@ -0,0 +1,13 @@
+# Break Statement
+
+`Break` statements can be used to terminate the execution within a nearest enclosing
+`while` and `foreach` statement.
```

Comment:
```suggestion
A `break` statement can be used to terminate the execution within the nearest enclosing `while` or `foreach` statement.
```

Reply:
You can write the entire description in a single line unless you need them in separate paragraphs.  
If you want to start a new paragraph use a new line. If we add new lines for each and every sentence they'll be rendered as if they are different paragraphs on the website.